### PR TITLE
CONTRIBUTING FILE 

### PR DESCRIPTION
**Pull request by akrish4**
Fixed : #3

### CODE REVIEW

**Comment by akrish4** [comment]
Kindly do check and merge it @praveenscience 
Kindly do add the same labels to PR also

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<h1> CONTRIBUTING ✨</h1>
```
Comment (praveenscience) [inline CONTRIBUTING.md:1]: Please remove all HTMLs and use Markdown.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+- When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a      change.
```
Comment (praveenscience) [inline CONTRIBUTING.md:3]: Why space?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- Please note we have a code of conduct, please follow it in all your interactions with the project and don't ping for approvals to the maintainers 
+
+## 🤝***First time contributing? We will help you out.*** 👍
+
+![GitHub custom open for collaboration](https://img.shields.io/badge/Open%20For-Collaboration-brightgreen?style=for-the-badge)
+
+Refer to the following articles on the basics of Git and Github and can also contact the Project Mentors, in case you are stuck:
+
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:12]: Remove spaces.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+- [Getting started with Git and GitHub](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github)
+- [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+- [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
+- [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
+
+***If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).***
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:19]: Remove empty lines.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+- Take a look at the Existing [Issues](https://github.com/praveenscience/Internship-LMS-FrontEnd/issues) or create your own Issues!
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Fork the Repo and create a Branch for any Issue that you are working upon.
+- Read the [Code of Conduct](https://github.com/praveenscience/Internship-LMS-FrontEnd/blob/main/CODE_OF_CONDUCT.md)
+- Watch all these  [videos](https://www.youtube.com/playlist?list=PLO_Y0rsm7b3Yc8FtNhCuD5TO0QM8CnTZj) before contributing .
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+- Add Screenshots to help us know what this Script is all about.
+
+
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:35]: Remove empty lines.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+## Contribution Guidelines🏗
+
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and make it better. 
+To start contributing, follow the below guidelines: 
+
+**1.** Fork [this](https://github.com/praveenscience/Internship-LMS-FrontEnd) repository.
+Click on the <a href="https://github.com/praveenscience/Internship-LMS-FrontEnd"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
```
Comment (praveenscience) [inline CONTRIBUTING.md:42]: Use Markdown.

Comment (praveenscience) [reply CONTRIBUTING.md:42]: Add ordered list in the right order.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+**2.** Clone the forked repository. Open terminal and type:
+
+```bash
+git clone https://github.com/<your-github-username>/Internship-LMS-FrontEnd.git
```
Comment (praveenscience) [inline CONTRIBUTING.md:47]: Add username.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+```
+
+**3.** Navigate to the project directory.
+
+```bash
+cd Internship-LMS-FrontEnd
+```
+**4.** Add a reference(remote) to the original repository.
```
Comment (praveenscience) [inline CONTRIBUTING.md:55]: Spaces are inconsistent.

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+```bash
+git remote add upstream https://github.com/praveenscience/Internship-LMS-FrontEnd.git
```
Comment (praveenscience) [inline CONTRIBUTING.md:57]: Add username.

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+
+**4.** Make a feature branch
+```bash
+git checkout -b <branch-name>
+
+```
+**5.**  Perfom your desired changes to the code base.
+
+
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:77]: Remove empty lines.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+```
+**9.** To create a pull request, click on `compare and pull requests`. Please ensure you compare your feature branch to the desired branch of the repo you are suppose to make a PR to.
+
+**10.** Add appropriate title and description to your pull request explaining your changes and efforts done ✍
+
+**11.** Click on `Create Pull Request` ([PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)) ! 👍
+
+**12.** **Congratulations!** Sit and relax, you've made your contribution to [Internship-LMS-FrontEnd](https://github.com/praveenscience/Internship-LMS-FrontEnd) project.
+
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:102]: Remove empty lines.

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+***:trophy: After this, project leaders and mentors will review the changes and will merge your PR if they are found good, otherwise we will suggest the required changes.***
+
+
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:106]: Remove empty lines.

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+**9.** To create a pull request, click on `compare and pull requests`. Please ensure you compare your feature branch to the desired branch of the repo you are suppose to make a PR to.
+
+**10.** Add appropriate title and description to your pull request explaining your changes and efforts done ✍
+
+**11.** Click on `Create Pull Request` ([PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)) ! 👍
+
+**12.** **Congratulations!** Sit and relax, you've made your contribution to [Internship-LMS-FrontEnd](https://github.com/praveenscience/Internship-LMS-FrontEnd) project.
+
+
+***:trophy: After this, project leaders and mentors will review the changes and will merge your PR if they are found good, otherwise we will suggest the required changes.***
```
Comment (praveenscience) [inline CONTRIBUTING.md:103]: Why so much bold and italic here?

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+For major changes, you are welcomed to open an issue  about what you would like to contribute. Enhancements will be appreciated.
+
+#### All the Best!🥇
+
+<p align = "center">
+
+<a href="https://github.com/akrish4"><img src="http://ForTheBadge.com/images/badges/built-by-developers.svg" alt="built by developers"></a>
+[![built with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://github.com/praveenscience/Internship-LMS-FrontEnd)
+
+</p>
```
Comment (praveenscience) [inline CONTRIBUTING.md:117]: Remove HTML and use only Markdown.

File: ReadMe.md [hunk 15]
```diff
@@ -107,3 +107,7 @@ Please check the contributing guidelines of my previous project, and it's going
 * Praveen Kumar Purushothaman ([Website](https://praveen.science/) &#1421; [LinkedIn](https://www.linkedin.com/in/praveentech/) &#1421; [YouTube](https://www.youtube.com/praveenscience?sub_confirmation=1))
 * Manju Ramachandran ([LinkedIn](https://www.linkedin.com/in/manjuramachandran10/))
 * Saloni Sawarkar ([LinkedIn](https://www.linkedin.com/in/saloni-sawarkar/))
+
+
```
Comment (praveenscience) [inline ReadMe.md:111]: Remove empty lines.

File: ReadMe.md [hunk 16]
```diff
@@ -107,3 +107,7 @@ Please check the contributing guidelines of my previous project, and it's going
...
+## 💥 How to Contribute ?
+- If you wish to contribute kindly check the [CONTRIBUTING.md](https://github.com/praveenscience/Internship-LMS-FrontEnd/blob/main/CONTRIBUTING.md)🤝
```
Comment (praveenscience) [inline ReadMe.md:113]: Add empty line. Remove List item.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please fix the issues.

File: ReadMe.md (hunk 16 above)
Comment (akrish4) [reply ReadMe.md]: updated everything according to the above mentioned @praveenscience

**Comment by akrish4** [comment]
Resolved all above issues @praveenscience

File: CONTRIBUTING.md (hunk 2 above)
Comment (praveenscience) [reply CONTRIBUTING.md:2]: Need line space, why space before `change`?

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+```bash
+git pull upstream main 
+```
+**7.** Make a feature branch
+```bash
+git checkout -b <branch-name>
+
+```
+**8.**  Perfom your desired changes to the code base.
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:65]: Line Spaces - remove.

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+git remote -v
+```
+**6.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository)
+```bash
+git pull upstream main 
+```
+**7.** Make a feature branch
+```bash
+git checkout -b <branch-name>
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:62]: Line Spaces - remove.

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+```
+**8.**  Perfom your desired changes to the code base.
+
+**9.** Track your changes✔️
+```bash
+#Add changes to Index
+git add .
+```
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:71]: Line Spaces - remove.

File: CONTRIBUTING.md [hunk 20]
```diff
@@ -0,0 +1,96 @@
...
+**10.** Commit to the local repo
+```bash
+git commit -m "<your_commit_message>"
+```
+**11.** Push the committed changes in your feature branch to your remote repo
+```bash
+git push -u origin <brach-name>
+```
+**12.** To create a pull request, click on `compare and pull requests`. Please ensure you compare your feature branch to the desired branch of the repo you are suppose to make a PR to.
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:81]: Line Spaces - remove.

File: CONTRIBUTING.md [hunk 21]
```diff
@@ -0,0 +1,96 @@
...
+**13.** Add appropriate title and description to your pull request explaining your changes and efforts done ✍
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:83]: Line Spaces - remove.

File: CONTRIBUTING.md [hunk 22]
```diff
@@ -0,0 +1,96 @@
...
+**14.** Click on `Create Pull Request` ([PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)) ! 👍
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:85]: Line Spaces - remove.

File: CONTRIBUTING.md [hunk 23]
```diff
@@ -0,0 +1,96 @@
...
+**15.** **Congratulations!** Sit and relax, you've made your contribution to [Internship-LMS-FrontEnd](https://github.com/praveenscience/Internship-LMS-FrontEnd) project.
+
+**:trophy: After this, project leaders and mentors will review the changes and will merge your PR if they are found good, otherwise we will suggest the required changes.**
+
+## 💥 Issues:
+For major changes, you are welcomed to open an issue  about what you would like to contribute. Enhancements will be appreciated.
+
+#### All the Best!🥇
+
+[![fork](http://ForTheBadge.com/images/badges/built-by-developers.svg")](https://github.com/akrish4)
```
Comment (praveenscience) [inline CONTRIBUTING.md:95]: Wrong link and syntax?

File: ReadMe.md [hunk 24]
```diff
@@ -107,3 +107,7 @@ Please check the contributing guidelines of my previous project, and it's going
 * Praveen Kumar Purushothaman ([Website](https://praveen.science/) &#1421; [LinkedIn](https://www.linkedin.com/in/praveentech/) &#1421; [YouTube](https://www.youtube.com/praveenscience?sub_confirmation=1))
 * Manju Ramachandran ([LinkedIn](https://www.linkedin.com/in/manjuramachandran10/))
 * Saloni Sawarkar ([LinkedIn](https://www.linkedin.com/in/saloni-sawarkar/))
+
+## 💥 How to Contribute ?
+
+If you wish to contribute kindly check the [CONTRIBUTING.md](https://github.com/praveenscience/Internship-LMS-FrontEnd/blob/main/CONTRIBUTING.md)🤝
```
Comment (praveenscience) [inline ReadMe.md:113]: Period after the sentence. Space after emoji?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please fix the issues.

File: CONTRIBUTING.md (hunk 2 above)
Comment (akrish4) [reply CONTRIBUTING.md]: updated

**Comment by akrish4** [comment]
Updated all issues @praveenscience

File: CONTRIBUTING.md [hunk 25]
```diff
@@ -0,0 +1,91 @@
...
+
+## Contribution Guidelines🏗
+
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and make it better. 
+To start contributing, follow the below guidelines: 
+
+**1.** Fork [this](https://github.com/praveenscience/Internship-LMS-FrontEnd) repository.
+Click on the [![fork](https://img.icons8.com/ios/24/000000/code-fork.png")](https://github.com/praveenscience/Internship-LMS-FrontEnd) symbol at the top right corner.
+
+
```
Comment (praveenscience) [inline CONTRIBUTING.md:38]: Remove empty lines. Not consistent with rest of the document.

File: CONTRIBUTING.md [hunk 26]
```diff
@@ -0,0 +1,91 @@
...
+- Watch all these  [videos](https://www.youtube.com/playlist?list=PLO_Y0rsm7b3Yc8FtNhCuD5TO0QM8CnTZj) before contributing .
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+- Add Screenshots to help us know what this Script is all about.
+
+## Contribution Guidelines🏗
+
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and make it better. 
+To start contributing, follow the below guidelines: 
+
+**1.** Fork [this](https://github.com/praveenscience/Internship-LMS-FrontEnd) repository.
```
Comment (praveenscience) [inline CONTRIBUTING.md:35]: Not the right way of creating "Ordered Lists".

File: ReadMe.md [hunk 27]
```diff
@@ -107,3 +107,7 @@ Please check the contributing guidelines of my previous project, and it's going
 * Praveen Kumar Purushothaman ([Website](https://praveen.science/) &#1421; [LinkedIn](https://www.linkedin.com/in/praveentech/) &#1421; [YouTube](https://www.youtube.com/praveenscience?sub_confirmation=1))
 * Manju Ramachandran ([LinkedIn](https://www.linkedin.com/in/manjuramachandran10/))
 * Saloni Sawarkar ([LinkedIn](https://www.linkedin.com/in/saloni-sawarkar/))
+
+## 💥 How to Contribute ?
```
Comment (praveenscience) [inline ReadMe.md:111]: Remove space before `?`.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 25 above)
Comment (akrish4) [reply CONTRIBUTING.md:38]: actually these spaces won't affect the final outcome , only while reviewing it , ie the raw file , you will find spaces nut in the final outcome there won't be any spaces

Comment (praveenscience) [reply CONTRIBUTING.md:38]: Yes, I am aware of it.

Comment (akrish4) [reply CONTRIBUTING.md:38]: Then why are you always mentioning the extra space or empty line as an issue ?
It doesn't affect at all

File: CONTRIBUTING.md (hunk 26 above)
Comment (akrish4) [reply CONTRIBUTING.md:35]: I am only aware of this ordered list , let me know what is the right one then

File: CONTRIBUTING.md (hunk 25 above)
Comment (praveenscience) [reply CONTRIBUTING.md:38]: Yes, I am aware, it's the code standards I follow. 😁

File: CONTRIBUTING.md (hunk 17 above)
Comment (praveenscience) [reply CONTRIBUTING.md:64]: This definitely affects.

File: ReadMe.md (hunk 24 above)
Comment (praveenscience) [reply ReadMe.md:113]: Not resolved. Why did you mark as resolved?

File: CONTRIBUTING.md (hunk 26 above)
Comment (praveenscience) [reply CONTRIBUTING.md:35]: And don't resolve the conversation if you haven't made any changes. Please refer [Ordered Lists](https://www.markdownguide.org/basic-syntax/#ordered-lists).

File: CONTRIBUTING.md [hunk 28]
```diff
@@ -0,0 +1,91 @@
+# CONTRIBUTING ✨
+- When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
```
Comment (praveenscience) [inline CONTRIBUTING.md:2]: Need a line space before starting a UL for consistency.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please fix the issues.

**Comment by ssg8288** [comment]
Dear @akrish4,
I am a repo maintainer from dwoc and it is brought to our notice that you are repeatedly asking other fellow maintainers to credit you the label tags based on your choice and I have already warned you for this action and here in this repo you are again arguing with the fellow maintainer who is one of our eminent members from dwoc. It is your duty as a contributor to addressing the issue what a maintainer is pointing out because that will lead to points.
Also, most of us (maintainers) are actually discussing to ban you from dwoc as you are pulling up bulk of issues and PR for some worthless contributions just to top the leaderboard and ruin the chances of the ones of those actually contributing something.
You on the other hand after leading the leaderboard just by working on markdown still are not able to learn all markdown syntaxes lmao XD.
So, Kindly work as the maintainer says else we would be pleasured to ban you and kindly make some valuable contributions.

Issues faced in this repo:
- Grammatical Issue
- Consistency
- Markdown Syntax
look after that and stop copy-pasting stuff from one repo to another.

**Comment by VedantKhairnar** [comment]
I am Vedant, speaking as the Founder of DevScript and Organiser of DWOC.
@akrish4 you need to follow all rules provided by the maintainer without asking a question as he is one of the god level people we have met and is experienced than everyone of us. As a contributor, you are representing DWOC to the maintainer. Don't resolve the conversation if you haven't made any changes. `Consider this as a ban warning`. If any sort of complaint is brought to us against you, we have full power to ban you from the program.

**Comment by akrish4** [comment]
> I am Vedant, speaking as the Founder of DevScript and Organiser of DWOC.
> @akrish4 you need to follow all rules provided by the maintainer without asking a question as he is one of the god level people we have met and is experienced than everyone of us. As a contributor, you are representing DWOC to the maintainer. Don't resolve the conversation if you haven't made any changes. `Consider this as a ban warning`. If any sort of complaint is brought to us against you, we have full power to ban you from the program.

I am sorry @VedantKhairnar @praveenscience  , I just asked a doubt regarding `spaces` to maintainer , I will resolve it and I have resolved  the conversation by mistake , I will make it done

**Comment by akrish4** [comment]
> Dear @akrish4,
> I am a repo maintainer from dwoc and it is brought to our notice that you are repeatedly asking other fellow maintainers to credit you the label tags based on your choice and I have already warned you for this action and here in this repo you are again arguing with the fellow maintainer who is one of our eminent members from dwoc. It is your duty as a contributor to addressing the issue what a maintainer is pointing out because that will lead to points.
> Also, most of us (maintainers) are actually discussing to ban you from dwoc as you are pulling up bulk of issues and PR for some worthless contributions just to top the leaderboard and ruin the chances of the ones of those actually contributing something.
> You on the other hand after leading the leaderboard just by working on markdown still are not able to learn all markdown syntaxes lmao XD.
> So, Kindly work as the maintainer says else we would be pleasured to ban you and kindly make some valuable contributions.
> 
> Issues faced in this repo:
> 
> * Grammatical Issue
> * Consistency
> * Markdown Syntax
>   look after that and stop copy-pasting stuff from one repo to another.

Hi @ssg8288 
I didn't even argue with maintainer , kindly don't post fake allegations against me , I just asked a question to maintainer regarding spaces and syntax , 
***now pls don't take this comment as a argument against you*** 
this is not arguing actually , this is conversation between a participant and contributor regarding errors so I just asked him a question and I was clearing my doubt 

And copy paste is a normal thing which is acceptable because this is open source , 
and I can open bulk of issues as it is mentioned that we can work on multiple issues at same time , 
and yes i was leading in leaderboard with markdown things and yes I admit I am not expert in it , I am still learning , Did I say that I am expert in markdown ? 

And if I open issues for markdown and you think it is worthless? then why will maintainers assign me ? they think it is needed for their project so they assign , they could close it 

***now pls don't thinl that I am arguing with you *** 
thank you

**Comment by ssg8288** [comment]
> 
> 
> > Dear @akrish4,
> > I am a repo maintainer from dwoc and it is brought to our notice that you are repeatedly asking other fellow maintainers to credit you the label tags based on your choice and I have already warned you for this action and here in this repo you are again arguing with the fellow maintainer who is one of our eminent members from dwoc. It is your duty as a contributor to addressing the issue what a maintainer is pointing out because that will lead to points.
> > Also, most of us (maintainers) are actually discussing to ban you from dwoc as you are pulling up bulk of issues and PR for some worthless contributions just to top the leaderboard and ruin the chances of the ones of those actually contributing something.
> > You on the other hand after leading the leaderboard just by working on markdown still are not able to learn all markdown syntaxes lmao XD.
> > So, Kindly work as the maintainer says else we would be pleasured to ban you and kindly make some valuable contributions.
> > Issues faced in this repo:
> > 
> > * Grammatical Issue
> > * Consistency
> > * Markdown Syntax
> >   look after that and stop copy-pasting stuff from one repo to another.
> 
> Hi @ssg8288
> I didn't even argue with maintainer , kindly don't post fake allegations against me , I just asked a question to maintainer regarding spaces and syntax ,
> _**now pls don't take this comment as a argument against you**_
> this is not arguing actually , this is conversation between a participant and contributor regarding errors so I just asked him a question and I was clearing my doubt
> 
> And copy paste is a normal thing which is acceptable because this is open source ,
> and I can open bulk of issues as it is mentioned that we can work on multiple issues at same time ,
> and yes i was leading in leaderboard with markdown things and yes I admit I am not expert in it , I am still learning , Have I ever told you that I am expert in markdown ?
> 
> And if I open issues for markdown and you think it is worthless? then why will maintainers assign me ? they think it is needed for their project so they assign , or else they could close it
> 
> _**now pls don't thinl that I am arguing with you**_
> thank you

Yes Yes Yes,  I'm not thinking that you are arguing as this is a conversation going on and neither have I said that you told me you're an expert and Yes Yes Yes, I'm not accusing you of any false allegations I'm speaking what other's are saying if you don't trust me then you will soon get a proof of it if you wish so. 
The main issue why everyone is frustrated here is because you are pulling issues for welcome bots, and several noncountable issues also asking the maintainers to award you with labels because that will do good to you isn't it see this below have attached a reference for you :

1)
image

many maintainers have a complaint regarding the same against you

2)
image

it is totally dependent on the maintainer what he wants to award you can't ask him.

Also, for your question where you asked why maintainers assign you certain issues? because here almost everyone is new and certain maintainers don't know how the things work and we all are guiding them. Also, if you have been assigned an issue it must be kept in mind what are the maintainer's requirements and should not have any sort of plagiarism involved.


For the point, I said you are topping the leader board .....because you are just working on markdown and still you haven't got your basics cleared for the basic syntaxes. Until now someone would have at least mastered that art and amount of practice this is why copy and pasting is not a good practice here pal.

And if you would have been totally right on your path the founder wouldn't have to make an effort to approach and talk to you.

Yes yes yes I'm still not thinking this is an argument and we are talking peacefully.
@VedantKhairnar @praveenscience @Zircoz
Thank you.

**Comment by akrish4** [comment]
yes @ssg8288 
I admit I have asked better labels but I didn't force anyone 
I wanted a better better label so I asked , if they feel that issue needs  they will add a better label 
Its mentioned that maintainer will decide , yes maintainer is only deciding , I have  just asked thats all 
And now a days I am not asking any better labels

**Pull request closed without merging**